src/Veldrid/OpenGL/OpenGLGraphicsDevice.cs

```diff
@@ -955,100 +981,113 @@ private void ExecuteWorkItem(ExecutionThreadWorkItem workItem)
                     switch (workItem.Type)
                     {
                         case WorkItemType.ExecuteList:
-                        {
-                            OpenGLCommandEntryList list = (OpenGLCommandEntryList)workItem.Object0;
-                            try
                             {
-                                list.ExecuteAll(_gd._commandExecutor);
-                                list.Parent.OnCompleted(list);
-                            }
-                            finally
-                            {
-                                if (!_gd.CheckCommandListDisposal(list.Parent))
+                                OpenGLCommandEntryList list = (OpenGLCommandEntryList)workItem.Object0;
```

Comment:
The indentation of these "case" blocks got changed in the PR -- can you revert that? There's an option in VS for not indenting blocks inside cases:

![image](https://user-images.githubusercontent.com/8918977/40891664-cfc0236c-673e-11e8-9635-b53ba7ff6d10.png)

Reply:
Weird, I couldn't see these changes in VS (and I still can't)

Reply:
The blocks under the `case` statements are indented, when they shouldn't be. If you uncheck that option, then auto-formatting the document should fix it (does for me, at least)